nixos/modules/services/web-apps/nextcloud.nix

```
@@ -485,9 +485,8 @@ in {
           wantedBy = [ "multi-user.target" ];
           before = [ "phpfpm-nextcloud.service" ];
           path = [ occ ];
+          serviceConfig.ExecStartPre = "+${pkgs.coreutils}/bin/install -m750 --owner nextcloud --group nextcloud -d ${cfg.home}";
```

I guess this should be enough?

```suggestion
          serviceConfig.ExecStartPre = "+${pkgs.coreutils}/bin/install -m710 --owner nextcloud --group nextcloud -d ${cfg.home}";
```